File: Example/PDStrategy/PDStrategy/PDStrategy/Views/PDActivityIndicatorView.m
//
//  PDActivityIndicatorView.m
//  PDStrategy
//
//  Created by Pavel Deminov on 19/11/2017.
//  Copyright © 2017 Pavel Deminov. All rights reserved.
//

#import "PDActivityIndicatorView.h"

@implementation PDActivityIndicatorView

- (instancetype)initWithActivityIndicatorStyle:(UIActivityIndicatorViewStyle)style {
    self = [super initWithActivityIndicatorStyle:style];
    [self setup];
    return self;
}

- (void)setup {
    self.translatesAutoresizingMaskIntoConstraints = NO;
    self.hidesWhenStopped = YES;
}

@end

File: Example/PDStrategy/PDStrategy/PDStrategy/Views/PDActivityIndicatorView.swift
//
//  PDActivityIndicatorView.swift
//  PDStrategy
//
//  Created by Pavel Deminov on 05/11/2017.
//  Copyright © 2017 Pavel Deminov. All rights reserved.
//

import UIKit

class PDActivityIndicatorView: UIActivityIndicatorView {

    var x: NSLayoutConstraint?
    var y: NSLayoutConstraint?
    
    required init(coder: NSCoder) {
        super.init(coder: coder)
        self.setup()
    }
    
    override init(activityIndicatorStyle style: UIActivityIndicatorViewStyle) {
        super.init(activityIndicatorStyle: style)
        self.setup()
    }
    
    func setup() {
        translatesAutoresizingMaskIntoConstraints = false
        hidesWhenStopped = true
    }
}
